tests/conftest.py

```diff
@@ -798,6 +799,22 @@ def polarion_testsuite_properties(record_testsuite_property, pytestconfig):
     )
 
 
+@pytest.fixture(scope='function', autouse=True)
+def health_checker(request):
+    node = request.node
+    # Limit the health check for tier4a, tier4b, tier4c
+    tier4_marks = [m.mark for m in [tier4a, tier4b, tier4c]]
+    for mark in node.iter_markers():
+        if mark in tier4_marks:
+            log.info("Checking for Ceph Health OK ")
+            status = ceph_health_check()
+            if status:
+                log.info("Health check passed")
+                return
+            else:
+                pytest.skip("Ceph Health check failed")
```

Comment:
why would we need to skip the test if ceph health is not OK. I believe we need to raise assert here.